app/controllers/works_controller.rb

```diff
@@ -3,9 +3,15 @@
 class WorksController < ApplicationController
   before_action :set_work, :check_auth
 
-
   def show
-
+    respond_to do |format|
+      format.html
+      format.ris {
+        send_data RisSerializer.new(@work).to_ris,
+          disposition: 'inline',
+          type: "application/x-research-info-systems"
```

Comment:
So I don't believe this should be disposition 'inline' -- that tells the browser to try to display it in a  browser tab if it can. Probably no browsers can, but still not actually what we want to be requesting. 

The code in chf_sufia is [here](https://github.com/sciencehistory/chf-sufia/blob/a103c5908af90c9076cb826140527213ba2e645f/app/controllers/curation_concerns/generic_works_controller.rb#L31-L34)

One thing we can see it is doing, is it IS specifying a nice filename to the browser to use for 'save as', instead of just guessing from the URL (which will end up the friendly_id as the filename). 

We haven't conquered that yet in this app, so don't need to in order to merge this PR -- but how about you leave us a GH issue reminding us that we need to fix the filename of the RIS download, and linking to the relevant code location to start with in chf_sufia as above?

If we look through further to the [encoding_safe_content_disposition method](https://github.com/sciencehistory/chf-sufia/blob/master/app/helpers/application_helper.rb#L31) , we see it is setting content-disposition to "attachment", not "inline". So let's do that here now, since that's easy. 

(Don't try to copy the logic for escaping or creating filename from chf_sufia -- there's a better way to do it, but we don't have to do it now to merge this PR! Let's leave an Issue reminding us to do it later as above).

Reply:
I also believe you don't have to supply `type` -- I think the fact that `format.ris` is there is enough to have Rails look up the right type we already registered for `ris`, no need to specify it again non-DRY. 

I could be wrong, but you can try it and the tests should tell us if it worked!

Reply:
Looks like you do need type -- it serves it as the default (application/octet-stream) otherwise.

Reply:
That's annoying. Hmm, what if you put `type: :ris`? Or we could research Rails more, I feel like there's a way to do this where it uses the type you specified as responding to.

Reply:
That worked!